.github/actions/build-test/ubuntu-i386/action.yml

```
@@ -75,6 +75,19 @@ runs:
           --directory ${install_dir}
         echo "${install_dir}/bin" >> $GITHUB_PATH
 
+    - name: Install `gh` CLI
+      shell: ${{ env.shell }}
+      run: |
+        # https://github.com/cli/cli/blob/trunk/docs/install_linux.md#debian-ubuntu-linux-raspberry-pi-os-apt
+        (type -p wget >/dev/null || (apt update && apt-get install wget -y)) \
+          && mkdir -p -m 755 /etc/apt/keyrings \
+                && out=$(mktemp) && wget -nv -O$out https://cli.github.com/packages/githubcli-archive-keyring.gpg \
+                && cat $out | tee /etc/apt/keyrings/githubcli-archive-keyring.gpg > /dev/null \
+          && chmod go+r /etc/apt/keyrings/githubcli-archive-keyring.gpg \
+          && echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+          && apt update \
```

```suggestion
          && apt update \
```

When looking at your shared run build [log](https://github.com/hazelcast/hazelcast-cpp-client/actions/runs/13507682122/job/37740965364) I see this causes a lot of packages to be updated and installed causing a lot of installetions. Do we really need other updating for installing `gh`?

---

> When looking at your shared run build [log](https://github.com/hazelcast/hazelcast-cpp-client/actions/runs/13507682122/job/37740965364) I see this causes a lot of packages to be updated and installed causing a lot of installetions. Do we really need other updating for installing `gh`?

Are you looking at the right `apt` section?

Looking at the log:
```
apt-get install -y build-essential cmake curl git libssl-dev maven net-tools openjdk-17-jre-headless gdb curl
{...}
6 upgraded, 202 newly installed, 0 to remove and 50 not upgraded.
```
_(existing)_

```
apt-get install wget -y
{...}
apt install gh -y
{...}
The following NEW packages will be installed:
wget
0 upgraded, 1 newly installed, 0 to remove and 50 not upgraded.
The following NEW packages will be installed:
gh
0 upgraded, 1 newly installed, 0 to remove and 50 not upgraded.
```
_(new)_

---

correct, i looked at the `apt-get`.